DockerFiles/run_kasm_ros.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+# Configurable variables
+DOCKER_IMAGE=62eb1ab76978
+
+WEB_PORT=10334
+SSH_PORT=7901
+CONTAINER_NAME=ff_ros2_DISCOWER_container
+VNC_PASSWORD=password
+# NOETIC_BASE_DIR=${HOME}/workspace/ros_noetic
+
+# Check if a container with the same name already exists
+if [ $(docker ps -aq -f name=^${CONTAINER_NAME}$) ]; then
+    echo "A container with the name $CONTAINER_NAME already exists."
+    # Choose what to do here: stop/remove the existing container, or exit
+    # docker stop $CONTAINER_NAME
+    # docker rm $CONTAINER_NAME
+    # or
+    exit 1
+fi
+
+# Run the Docker container
+docker run -itd \
+      --shm-size 32g \
+      --runtime=nvidia \
```

Comment:
Does this mean that we have to have an nvidia gpu? Probably just check whether it exists before running it?

Reply:
good point. I will take this into account

Reply:
I've added a check for the existence of the nvidia-smi command.